tests/test_custom_logo_docs.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+
+import pytest
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+TEST_HTTP_FAVICON_URL = "http://favicon-url.com/favicon.png"
+TEST_HTTPS_FAVICON_URL = "http://favicon-url.com/favicon.png"
+DEFAULT_FASTAPI_FAVICON = "https://fastapi.tiangolo.com/img/favicon.png"
+
+
+def get_client_with_custom_docs_logo(favicon_url):
+    app = FastAPI(favicon_url=favicon_url)
+
+    @app.get("/items/")
+    async def read_items():
+        return {"id": "foo"}
+
+    return TestClient(app)
```

Comment:
It would be good to get rid of this, and use a pytest fixture. (I do not insist on that though)